src/codeql.ts

```diff
@@ -104,6 +104,8 @@ async function runQuery(
     outputSarif(
       codeql,
       bqrs,
+      nwo,
+      dbMetadata.creationMetadata?.sha || "HEAD",
```

Comment:
Could we pass this in as a `string?` instead and then not write the `revisionId` field if it isn't available? I think that would be a little nicer than writing `HEAD`. The extension could then take the decision to use `HEAD` in constructing the links if it doesn't find `revisionId` in the downloaded SARIF file.

Reply:
Thanks! Attempted to do this in https://github.com/dsp-testing/qc-run2/pull/655/commits/4b53aeeaeca3dbdbf8acd4a0bb09f9432c46d4e5 🙇🏽‍♀️